From ebfb955e19df9644afd7aeaf5ca7e6d11f51391c Mon Sep 17 00:00:00 2001
From: xiao lang <1115662687@qq.com>
Date: Mon, 6 May 2013 20:33:29 +0800
Subject: [PATCH 2/2] dwdw

---
 UserDAO.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/UserDAO.java b/UserDAO.java
index 8b85a94..9cc1a1f 100644
--- a/UserDAO.java
+++ b/UserDAO.java
@@ -15,7 +15,7 @@ public class UserDAO {
 	
 	Connection con = null;
 	PreparedStatement ps = null;
-	ResultSet rs = null;
+	ResultSet rs =;
 	
 	/**
 	 * 根据用户id返回一个用户对象，如果id不存在则返回null
-- 
1.8.0.msysgit.0

